scipy/optimize/_linprog_ip.py

```diff
@@ -841,7 +844,7 @@ def _linprog_ip(
         A=None,
         b=None,
         callback=None,
-        _T_o=[],
+        postsolve_args=(),
```

Comment:
Maybe keep this as None?

Reply:
I'm getting rid of inappropriate default values altogether. Not a complete overhaul, but a step in the right direction, I think.